ml-proto/spec/kernel.ml

```diff
@@ -82,9 +82,9 @@ and expr' =
   | Block of expr list                      (* execute in sequence *)
   | Loop of expr                            (* loop header *)
   | Break of var * expr option              (* break to n-th surrounding label *)
-  | Br_if of var * expr option * expr       (* conditional break *)
+  | Break_if of var * expr option * expr    (* conditional break *)
+  | Break_table of var list * expr option * expr  (* indexed break *)
```

Comment:
Naming these `Break_if` and `Break_table`, when the ast.ml names are `Br_if` and `Br_table`, makes it less obvious that kernel.ml is intended to be a subset of ast.ml.